site/content/reference/openshift-permissions.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Openshift Permissions"
+date: 2024-05-14T15:08:41-07:00
+# Change draft status to false to publish doc
+draft: false
+# Description
+# Add a short description (150 chars) for the doc. Include keywords for SEO.
+# The description text appears in search results and at the top of the doc.
+description: "Learn about the permissions given to the NGINX Gateway Fabric Pod in an Openshift environment."
+# Assign weights in increments of 100
+weight: 100
+toc: true
+tags: [ "docs" ]
+# Create a new entry in the Jira DOCS Catalog and add the ticket ID (DOCS-<number>) below
+docs: "DOCS-000"
+---
+
+## Overview
+
+To deploy NGINX Gateway Fabric on an Openshift environment, additional permissions are granted to the Pod which are defined
+in a SecurityContextConstraints object. This document attempts to describe the permissions given to the Pod and
+why they were given.
+
+
+## Specification
+
+{{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
```

Comment:
Do we want the specs to be in a table format? Or would it be better to split each field into a header and have an explanation like that?

Reply:
Check out https://docs.nginx.com/mesh/reference/permissions/ as one way we could do it (the capabilities are really the only thing in here, but we could do something similar with other fields).